### DISCUSSION TITLE
Invoicing Problem In Turkey Sellers ("3.0.0 General Release 2024-06-03" regarding the changes "Fulfillment-related changes in Turkey".)

### DISCUSSION THREAD

**Discussion by salyangoz** [etsy/open-api#1272] [Ideas]
Dear Etsy Support Team,

We have encountered an issue with customer address information transmission after your release "3.0.0 General Release 2024-06-03" regarding the changes "Fulfillment-related changes in Turkey". Since then we (yengec.co) cannot provide our invoicing solutions correctly for our Turkish sellers on Etsy. For legal side in Turkey, sellers must create invoices and we are using shipping informations for that. When you removed that information for this change, our system stopped working for whole sellers. There is no other way to create automatic invoices for sellers.

We request your guidance for a solution to that matter, so that both the invoicing and the shipment address details of Turkish sellers' orders are transmitted correctly to our integration.

This affected 100+ Etsy Sellers on Turkey that we serve. 

Also this affected for self shipping integration solutions as well. 

Our App Name : Yengeç

Thanks in advance for your help. @etsyachristensen, @nateswart 

Best regards

**Comment by turboturtleD** [comment]
You need to email Etsy support directly.  Best luck.

developer@etsy.com

**Reply by salyangoz** [reply]
> You need to email Etsy support directly. Best luck.
> 
> [developer@etsy.com](mailto:developer@etsy.com)

We already tried this option. However, unfortunately, we did not receive a satisfactory response, and we did not get an adequate answer regarding the critical nature of the issue.

**Comment by Omerbchvn** [comment]
Dear Etsy Support Team,
We have encountered an issue with customer address information transmission after your release "3.0.0 General Release 2024-06-03" regarding the changes "Fulfillment-related changes in Turkey". Since then we (yengec.co) cannot provide our invoicing solutions correctly for our Turkish sellers on Etsy. For legal side in Turkey, sellers must create invoices and we are using shipping informations for that. When you removed that information for this change, our system stopped working for whole sellers. There is no other way to create automatic invoices for sellers.
We request your guidance for a solution to that matter, so that both the invoicing and the shipment address details of Turkish sellers' orders are transmitted correctly to our integration.
This affected 100+ Etsy Sellers on Turkey that we serve.
Also this affected for self shipping integration solutions as well.
Our App Name : Yengeç https://app.yengec.co/
Best regards
